test_runner/.gitignore

```diff
@@ -8,3 +8,4 @@ src/test/kotlin/ftl/fixtures/error_result/*.xml
 src/test/kotlin/ftl/fixtures/success_result/*.xml
 *.json
 *-describe.adoc
+should_exists.txt
```

Comment:
what is this ?

Reply:
of all the weird quirks of windows. When you run test via gradle it adds this file. I just removed it via .gitignore.

Reply:
This should be removed at the end of some tests `UtilsTest`
```kotlin
companion object {
        @JvmStatic
        @AfterClass
        fun removeVerificationFile() {
            File(VERIFICATION_FILE).run {
                if (exists()) delete()
            }
        }
    }
```
Most likely test run terminated somewhere in the middle.
Just FYI